Swap iq dashboard

Emmanueldmlr opened the pull request:
# Swap IQ Dashboard

- [x] we need a proper icon for quick swap
- [x] add 2 new items at the end w proper links & icons: Binance & Upbit

## Linked issues

fixes https://github.com/EveripediaNetwork/issues/issues/767

kesar commented:
<img width="339" alt="Captura de Pantalla 2022-09-30 a las 18 00 17" src="https://user-images.githubusercontent.com/1288106/193310527-a3ea7ab5-c034-4176-a0c3-eae6f82b7b20.png">

pls could you fix this '  ? i added this bug :(

Emmanueldmlr commented:
> <img alt="Captura de Pantalla 2022-09-30 a las 18 00 17" width="339" src="https://user-images.githubusercontent.com/1288106/193310527-a3ea7ab5-c034-4176-a0c3-eae6f82b7b20.png">
> 
> pls could you fix this ' ? i added this bug :(

fixed now
